src/uct/ib/base/ib_md.c

```diff
@@ -1904,4 +1970,45 @@ uct_component_t uct_ib_component = {
     .flags              = 0,
     .md_vfs_init        = uct_ib_md_vfs_init
 };
-UCT_COMPONENT_REGISTER(&uct_ib_component);
+UCT_COMPONENT_REGISTER_DEF(&uct_ib_component, ib);
+
+static void uct_ib_md_ops_add(uct_ib_md_ops_entry_t *entry)
+{
+    uct_ib_md_ops_entry_t *p;
+
+    ucs_list_for_each(p, &uct_ib_md_ops_list, list) {
+        if (p->priority < entry->priority) {
```

Comment:
maybe arrange uct_ib_ops array according to priority? and remove uct_ib_md_ops_list altogether.